modules/tinymce/changelog.txt

```diff
@@ -1,6 +1,7 @@
 Version 5.5.0 (TBD)
     Added new `table_column_resizing` setting to control how the table is affected when resizing columns using the resize bars #TINY-6001
     Added ability to load TinyMCE in inline mode within a ShadowRoot #TINY-6128
+    Added the `image` `images_upload_handler` can now chose to have images removed on a failed upload #TINY-6011
```

Comment:
`image` shouldn't be surrounded by backticks here, as it's not referring to a setting/plugin name.